torch/onnx/_internal/fx/passes/decomp.py

```diff
@@ -32,6 +37,30 @@ def _rename_placeholder_targets(
     module.recompile()
 
 
+def wrapped_fn(fn, mode, allow_non_fake_inputs, args):
```

Comment:
Related to onnx error comment. Type annotating the arguments helps bring a lot clarity.